engine/runtime-with-instruments/src/test/scala/org/enso/interpreter/test/instrument/RuntimeErrorsTest.scala

```diff
@@ -1908,8 +1908,8 @@ class RuntimeErrorsTest
     context.receiveNIgnorePendingExpressionUpdates(
       3
     ) should contain theSameElementsAs Seq(
-      TestMessages.update(contextId, x1Id, ConstantsGen.NOTHING_BUILTIN),
-      TestMessages.update(contextId, mainRes1Id, ConstantsGen.NOTHING_BUILTIN),
+      TestMessages.update(contextId, x1Id, ConstantsGen.NOTHING),
+      TestMessages.update(contextId, mainRes1Id, ConstantsGen.NOTHING),
```

Comment:
@jdunkerley this change is definitely not obvious.

`Nothing` is a builtin type, and they have special handling in the compiler. When the program is compiled
1. All the builtins are loaded into the scope, making the type of `Nothing` `Standard.Builtins.Main.Nothing`
2. When the compiler encounters the `Nothing` definition in the standard library
    ```py
    @Builtin_Type
    type Nothing
    ```
    It shadows the builtin definition of `Nothing` with `Standard.Base.Nothing.Nothing` (the module where the new definition is)

Now returning to the test,
```py
      from Standard.Base.Errors.Common import all
      main =
          x = IO.println "MyError"
          x
```

The program imports `Standard.Base.Errors.Common` which in turn imports `project.Meta`, and this PR changes `Meta.enso` [test/micro-distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso](https://github.com/enso-org/enso/pull/7052/files#diff-58a13d09bdc0be0a325780561d499ede633fa76c12951e84f627f8389076e1e1) and adds
```diff
+from Standard.Base.Nothing import Nothing
+
```

Now the compiler sees the definition of `Nothing` in the standard library and shadows the builtin definition. And that's why the return type of `Nothing` changes in the test.